### PR TITLE
All the haml clean up

### PR DESCRIPTION
**Pull request by Br3nda**
i think this is the last of it!

### CODE REVIEW

File: app/views/devise/confirmations/new.html.haml [hunk 1]
```diff
@@ -1,17 +1,21 @@
 - content_for :title, "Resend confirmation instructions"
 
-= form_for(resource, :as => resource_name, :url => confirmation_path(resource_name), :html => { :method => :post, :class => 'form-horizontal', :role => 'form' }) do |f|
+= form_for(resource, as: resource_name,
+          url: confirmation_path(resource_name),
+          html: { method: :post,
+          class: 'form-horizontal',
```
Comment (pozorvlak) [inline app/views/devise/confirmations/new.html.haml]: This doesn't look right - shouldn't the `class` line up with `method`?

File: app/views/devise/passwords/new.html.haml [hunk 2]
```diff
@@ -1,15 +1,18 @@
 - content_for :title, "Forgot your password?"
 
-= form_for(resource, :as => resource_name, :url => password_path(resource_name), :html => { :method => :post, :class => 'form-horizontal', :role => 'form' }) do |f|
+= form_for(resource, as: resource_name,
+          url: password_path(resource_name),
+          html: { method: :post,
+          class: 'form-horizontal', role: 'form' }) do |f|
```
Comment (pozorvlak) [inline app/views/devise/passwords/new.html.haml]: Again, I think the alignment is wrong.

File: app/views/devise/registrations/_edit_password.html.haml [hunk 3]
```diff
@@ -1,23 +1,25 @@
-= form_for(resource, :as => resource_name, :url => registration_path(resource_name), :html => { :method => :put, :class => 'form-horizontal' }) do |f|
+= form_for(resource, as: resource_name,
+        url: registration_path(resource_name),
```
Comment (pozorvlak) [inline app/views/devise/registrations/_edit_password.html.haml]: Indentation here differs from the corresponding line in other files (eg `_edit_email.html.haml`): I don't know which one is right.

File: app/views/devise/sessions/new.html.haml [hunk 4]
```diff
@@ -1,17 +1,19 @@
 - content_for :title, "Sign in"
 
-= form_for(resource, :as => resource_name, :url => session_path(resource_name), :html => {:class => "form-horizontal"}) do |f|
+= form_for(resource, as: resource_name,
```
Comment (pozorvlak) [inline app/views/devise/sessions/new.html.haml]: Indentation different here to that on corresponding lines.

File: app/views/layouts/_meta.html.haml.bak [hunk 5]
```diff
@@ -0,0 +1,40 @@
+%head
```
Comment (pozorvlak) [inline app/views/layouts/_meta.html.haml.bak]: Looks like you committed this by accident.

File: app/views/members/_stats.html.haml [hunk 6]
```diff
@@ -2,34 +2,34 @@
 
 %ul.list-inline
   %li
-    - unless member.plantings.empty?
+    - if !member.plantings.empty?
```
Comment (pozorvlak) [inline app/views/members/_stats.html.haml]: I thought `unless` was preferred over `if !`...

Comment (maco) [inline app/views/members/_stats.html.haml]: ditto

File: app/views/layouts/_meta.html.haml.bak (hunk 5 above)
Comment (Br3nda) [inline app/views/layouts/_meta.html.haml.bak]: Ooooops

File: app/views/members/_stats.html.haml (hunk 6 above)
Comment (Br3nda) [inline app/views/members/_stats.html.haml]: Not when there is a else. 
Unless/else is confusing

**Comment by Br3nda** [comment]
Rebased, and edited git history to remove that .bak file.

**Comment by Br3nda** [comment]
@maco @pozorvlak ready for review again

File: app/views/devise/unlocks/new.html.haml [hunk 7]
```diff
@@ -1,10 +1,10 @@
 - content_for :title, "Resend unlock instructions"
 
 = form_for(resource, as: resource_name,
-                    url: unlock_path(resource_name),
-                    html: { method: :post,
-                    class: 'form-horizontal',
-                    role: 'form' }) do |f|
+    url: unlock_path(resource_name),
+    html: { method: :post,
```
Comment (pozorvlak) [inline app/views/devise/unlocks/new.html.haml]: The indentation's misleading here. I think you can fit the hash onto one line?

File: app/views/members/_stats.html.haml (hunk 6 above)
Comment (pozorvlak) [inline app/views/members/_stats.html.haml]: Cool, thanks - I was confused about why you were changing some `if !`s to `unless`es and some `unless`es to `if !`s.

File: app/views/devise/confirmations/new.html.haml [hunk 8]
```diff
@@ -1,17 +1,21 @@
 - content_for :title, "Resend confirmation instructions"
 
-= form_for(resource, :as => resource_name, :url => confirmation_path(resource_name), :html => { :method => :post, :class => 'form-horizontal', :role => 'form' }) do |f|
+= form_for(resource, as: resource_name,
+    url: confirmation_path(resource_name),
+    html: { method: :post,
+    class: 'form-horizontal',
```
Comment (pozorvlak) [inline app/views/devise/confirmations/new.html.haml]: Can you join the hash into one line? The indentation's currently misleading.

**Comment by Br3nda** [comment]
@pozorvlak all done

**Comment by Br3nda** [comment]
(super keen to get this wide ranging PR merged before it git conflicts)

**Pull request merged**